analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Branching.LearnConstraint.cs

```diff
@@ -482,11 +362,22 @@ public void Branching_LearnsObjectConstraint_TypePattern_Negated(string expressi
 }}
 Tag(""End"", arg);";
             var validator = SETestContext.CreateCS(code, ", object arg").Validator;
-            validator.ValidateTag("If", x => x.Should().BeNull("it could be null or any other type"));
-            validator.ValidateTag("Else", x => x.HasConstraint(ObjectConstraint.NotNull).Should().BeTrue());
-            validator.TagValues("End").Should().HaveCount(2)
-                .And.ContainSingle(x => x == null)
-                .And.ContainSingle(x => x != null && x.HasConstraint(ObjectConstraint.NotNull));
+            validator.ValidateContainsOperation(expectedOperation);
+            return validator;
+        }
+
+        private static ValidatorTestCheck CreateIfElseEndValidatorVB(string expression, OperationKind expectedOperation)
+        {
+            var code = @$"
+If {expression} Then
```

Comment:
Keep it in sync with the cs version and add
```
object isNull = null;
var isObject = new object();
```

Reply:
Not needed (yet). UTs for VB are not as intensive as C# ones. The idea is that most of it will just work thanks to C# effort and only very few stuff needs additional testing.